projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.scss

```diff
@@ -125,12 +125,73 @@
 		position: absolute;
 		right: 0;
 
+		@media (max-width: 600px) {
+			&.ab-top-menu > .menupop > .ab-sub-wrapper {
+				left: unset;
+			}
+		}
+
 		#wp-admin-bar-my-account {
 			.ab-item {
 				> .display-name {
 					color: currentColor;
 				}
 			}
+
+			#wp-admin-bar-my-wpcom-account {
+				padding: 4px 0;
+
+				.ab-item {
+					height: unset;
+					width: unset;
+				}
+
+				span.wpcom-button {
+					color: white;
+					background-color: #3858e9;
+					border-color: #3858e9;
+					border-radius: 2px;
+					padding: 0 14px;
+					box-sizing: border-box;
+					width: 100%;
```

Comment:
I'm seeing the button UI being off on the front end:

![Screenshot 2025-02-27 at 10 50 34 AM](https://github.com/user-attachments/assets/db969f15-f53c-42da-8202-be2a21370427)

Seems that the following CSS rule is not applied there.
![Screenshot 2025-02-27 at 10 54 43 AM](https://github.com/user-attachments/assets/915206e5-43ec-4eaf-841a-3fd90efda455)

Should we add `display: inline-block` here?

Reply:
Woops, good catch, it was because I changed `<button>` to `<span>` in the last minute (as button inside `a` is not valid HTML) and didn't check properly. It's fixed now!